### PR TITLE
Adding SBTB event

### PR DESCRIPTION
**Pull request by anicolaspp**

### CODE REVIEW

File: _events/2016-11-1-scala-by-the-bay.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+poster_hero: "/img/media/twitter-hq.jpg"
+poster_thumb: "/img/media/twitter-cabins_fe.jpg"
+
+location_section: true
+
+---
+
+## About the Conference
+
+Scala By the Bay is the yearly conference of SF Scala, the largest Scala meetup and community in the world, with the highest concentration of Scala developers and companies running Scala as a foundation of their global operations. The conference is held continuously since 2013, with all the videos published on functional.tv. In 2015, expanded with Twitter OSS conference, FinagleCon, and   bringing in a wealth of Twitter Stack, powering the global conversation, and Big Data Scala By the Bay  -- the synthesis of data engineering and data science with Apache Spark, Apache Kafka, and emerging streaming
```
Comment (adelbertc) [inline _events/2016-11-1-scala-by-the-bay.md]: extra space btwn "and" and "bringing"

Comment (anicolaspp) [inline _events/2016-11-1-scala-by-the-bay.md]: what is the problem with this part?

**Comment by milessabin** [comment]
This is just a general promo for SBTB isn't it? It's fine to promote any Typelevel specific activities and mention the main conference in that context, but I think this it a bit too generic to belong on this typelevel.org as it stands.

@adelbertc, were you coordinating a Typelevel hack day/unconference/track at SBTB? Maybe you could describe what people can expect from that and help @anicolaspp rework the post more along the lines of the [Lambda World one](http://typelevel.org/event/2016-09-conf-cadiz/)?

**Comment by adelbertc** [comment]
@milessabin No plans to coordinate a Typelevel track on my end. I remember we had entertained the idea of one but I think that fell through :/

**Comment by tpolecat** [comment]
Ok I was a little puzzled about this too. If there's no Typelevel event I don't think it really belongs on the website.

**Comment by durban** [comment]
Well, according to typelevel/general/issues/29 there _will_ be a typelevel track ... (if not, that issue probably should be closed).

**Comment by milessabin** [comment]
Hmm ... that's a bit of a shame. Seems like a bit of a missed opportunity given that many TL contributors will be there, I'll be keynoting etc. etc.

I had thought that @adelbertc was coordinating something, but I guess not. Can anyone else pick up where he left off?

**Comment by adelbertc** [comment]
Apologies if I gave the impression I was planning something - I recall we tried reaching out to Alexy but at some point it slipped through the cracks. If you think we still have time to plan something I'd love to help. Never planned anything like this before so I'm not sure where to begin, but tell me what to do and you have my assistance :-)

**Comment by adelbertc** [comment]
SBTB is over so probably no Typelevel event ;)

That being said we did have a lunch gathering today to plan the next Typelevel Summit so stay tuned for more info!

**Pull request closed without merging**